backend/src/database/seeds/01_capabilities.js

```diff
@@ -55,6 +55,14 @@ const capabilities = [
     id: 14,
     name: "picker_options_read_all",
   },
+  {
```

Comment:
I understand this, lets just be cautious how and when we group things in this bucket.